build-pipeline/vg-pipeline-module-pr.yml

```diff
@@ -131,6 +132,21 @@ stages:
             useLegacyPattern: false
             enableTransforms: false
             enableTelemetry: true
+        - task: replacetokens@3
+          displayName: "Token Replace .env.tmpl"
+          inputs:
+            targetFiles: './codepipes-ci-cd-modules/.env.tmpl => .env'
```

Comment:
won't this cause the .env writtin in line 101 to be overwritten?

Reply:
No, it seems to be working fine. weird syntax I know.
I tried `./codepipes-ci-cd-modules/.env.tmpl => ./codepipes-ci-cd-modules/.env` before but it resulted in `s/codepipes-ci-cd-modules/codepipes-ci-cd-modules/.env`.

https://dev.azure.com/cldcvrdevusa/Vanguard/_build/results?buildId=47244&view=logs&j=f833b1df-30e0-5a1f-cb52-9403a4b80eb5&t=39cde717-ce1c-5fa7-cee1-f797e53c9d8b&l=10

https://dev.azure.com/cldcvrdevusa/Vanguard/_build/results?buildId=47244&view=logs&j=f833b1df-30e0-5a1f-cb52-9403a4b80eb5&t=144b712f-7cf7-5412-f3e6-5b0dd728a381&l=10

Reply:
ah i see they are in different directories...